tests/e2e/lib.sh

```diff
@@ -905,17 +925,20 @@ wait_for_api() {
 }
 
 record_build_info() {
-    _record_build_info || {
+    _record_build_info "${1:-}" || {
```

Comment:
Even if it is a short function, could you add a local var for $1 ?

Reply:
done